docs/guides/insar_product_guide_template.md

```diff
@@ -0,0 +1,136 @@
+2. Images with identical incidence angles and beam mode
+3. Images with identical resolution and wavelength (usually from the same sensor)
+4. Images with the same viewing geometry (same path and frame)
+5. Images with identical polarizations (both HH or VV)
+
+In addition, the following suggestions may be helpful:
+
+1. Use images from similar seasons/growth/weather conditions
+2. For deformation mapping: limited spatial separation of acquisition locations (small physical baseline)
+3. For topographic mapping: limited time separation between images (small temporal baseline)
+
+To analyze deformation caused by a single discrete event, such as an earthquake, select images that bracket the event as closely in time as possible. Keeping the window narrowly focused on the time of the event will reduce the impacts of other processes that may mask the signal of the event of interest.
+
+{% block processing_options %}{% endblock %}
+
+{% block workflow %}{% endblock %}
+
+{% block packaging %}{% endblock %}
+
+## Limitations
+### Baseline Calculation
+The baseline is defined as the difference of the platform positions when a given area is imaged. HyP3 baselines are calculated using the best state vectors available. If precise orbits are not yet available for the input granules, restituted orbits will be used. The original predicted orbits are not used for InSAR processing in HyP3. **If no restituted or precise state vectors are available, the process will not run.**
+
+### Coherence
+The phase measurements in the two images used in InSAR must be coherent in order to detect change. Random changes in phase from one acquisition to the next can mask actual surface deformation. Vegetation is a common driver of decorrelation, as changes can easily take place in the interval between two acquisitions due to growth, seasonal changes, or wind effects. It will be difficult to generate valid interferograms with C-band data in heavily vegetated regions due to lack of coherence even with fairly short time intervals.
+
+Consider seasonality when selecting image pairs. Decorrelation can be particularly high when comparing phase from different seasons. Changes in the condition of vegetation (especially deciduous canopies), snow, moisture, or freeze/thaw state can impact phase measurements. In cases where a temporal baseline is required that spans seasons, it may be better to use an annual interferogram if possible so that the images are more comparable in terms of seasonality.
+
+{% block line_of_sight %}{% endblock %}
+
+{% block reference_point %}{% endblock %}
+
+## Error Sources
+On Demand InSAR products do not currently correct for some common sources of error in interferometry, such as atmospheric effects. Further processing or time series analysis can be performed by the user to identify or reduce the impact of some of these errors when using On Demand InSAR products for analysis.
+
+### Atmospheric Delay
```

Comment:
@forrestfwilliams From some of the comments I've heard David B. make, it sounds like maybe there are fewer concerns about some of the atmospheric impacts due to the smaller area covered by the bursts. Did I understand that correctly? Should we make amendments to these delay discussions for the burst products?

Reply:
When you're looking at small-scale deformation, tropospheric phase is less impactful but it is still definitely present. I think we should still discuss it here.

Reply:
Sounds good. I've made a [change to the common section](https://github.com/ASFHyP3/hyp3-docs/pull/360/commits/4761bd7b753c547d03cc3c16a58734af165b9a6f) to call out that it may be less of an issue for the bursts, but should still be considered. Does that look okay to you @forrestfwilliams? That section is currently common to both, and I figured it would be fine to keep it common, but add a mention to the difference between burst- and non-burst-based products.

Reply:
Hey @hjkristenson sorry if I was unclear, but I don't think we should treat tropospheric errors any differently for IW SLCs and bursts. Tropospheric effects are less impactful when looking at small scale deformation, but you can look at both small and large scale deformation in either a burst IFG or a IW SLC IFG. In other words, it's not the size of the interferogram that matters, it's the size of the deformation. I'll add an edit to make this clear.